health_check/contrib/twilio/backends.py

```diff
@@ -0,0 +1,34 @@
+import json
+import urllib.request
+
+from django.conf import settings
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceUnavailable, ServiceWarning
+
+
+class TwilioHealthCheck(BaseHealthCheckBackend):
+    def __init__(self):
+        super(TwilioHealthCheck).__init__()
+        self.services = getattr(self, 'services', [])
+
+    def check_status(self):
+        timeout = getattr(settings, 'HEALTHCHECK_TWILIO_TIMEOUT', 5)
+
+        try:
+            if len(self.services) > 0:
+                response = urllib.request.urlopen('https://gpkpyklzq55q.statuspage.io/api/v2/summary.json',
+                                                  timeout=timeout)
+                data = json.loads(response.read().decode('utf-8'))
```

Comment:
`json.load` excepts stream like the response object, so:
```python
json.load(response)
```
does the same thing, at least in Python 3.5 and up